src/NewTools-Core/StHeaderBar.class.st

```diff
@@ -65,7 +65,7 @@ StHeaderBar >> initializePresenters [
 		beNotHomogeneous;
 		borderWidth: 3;
 		spacing: 3;
-		add: (titleLabel := self newLabel) expand: false;
+		add: (titleLabel := self newLabel) expand: true;
```

Comment:
The expanding property is always true by default. So, there is no need to declare it. You can just say:
`add: (titleLabel := self newLabel);`